esmvaltool/recipes/recipe_climwip_test_basic.yml

```diff
@@ -41,11 +47,39 @@ preprocessors:
       scheme: linear
     mask_landsea:
       mask_out: sea
-    extract_region:
-      start_longitude: -10.0
-      end_longitude: 39.0
-      start_latitude: 30.0
-      end_latitude: 76.25
+
+    # -- cut region using corners --
+    # This option cuts rectangular regions by providing the corners
+    # extract_region:
+    #   start_longitude: -10.0
+    #   end_longitude: 39.0
+    #   start_latitude: 30.0
+    #   end_latitude: 76.25
+
+    # -- cut SREX regions --
+    # This option cuts one or more SREX regions
+    # extract_shape:
+    #   shapefile: shapefiles/srex.shp
+    #   method: contains
+    #   crop: true
+    #   ids:
+    #     - 'North Europe [NEU:11]'
+    #     - 'Central Europe [CEU:12]'
+    #     - 'South Europe/Mediterranean [MED:13]'
```

Comment:
please remove commented out blocks if not needed anymore or add explanation at the top of the commented out block when/why should be uncommented